common/src/main/java/io/quarkus/ts/openshift/common/actions/CopyLogsOnOpenShiftFailureActionImpl.java

```diff
@@ -0,0 +1,42 @@
+package io.quarkus.ts.openshift.common.actions;
+
+import io.fabric8.kubernetes.api.model.Pod;
+import io.quarkus.ts.openshift.common.Command;
+import io.quarkus.ts.openshift.common.config.Config;
+import io.quarkus.ts.openshift.common.injection.TestResource;
+import io.quarkus.ts.openshift.common.util.OpenShiftUtil;
+
+import java.io.File;
+
+public class CopyLogsOnOpenShiftFailureActionImpl implements OnOpenShiftFailureAction {
+
+    private static final String INSTANCES_LOGS_OUTPUT_DIRECTORY = "instance-logs";
```

Comment:
@Ladicek wrote `The config key must be ts.*. The config system will ignore keys that don't begin with ts..`
You probably forgot about it.

README should be updated to include info about this property

Reply:
It's in the `Config` javadoc.

Reply:
exactly

Reply:
Good point @rsvoboda . Added.